content/en/developers/estuary.md

```diff
@@ -2,8 +2,10 @@
 title: "Estuary"
 description: "Estuary makes storing data with Filecoin simpler by managing replication tasks and automating much of the workflow, allow you to carry on building your applications."
 menu:
-    build:
-        parent: "build-tools"
+    developers:
+        parent: "developers-tools"
+aliases:
+    - /build/estuary
```

Comment:
Are we still keeping these aliases as /build/xxx?

Reply:
Yeah. They're necessary so that if someone visits docs.filecoin.io/build/something then they get redirected to the correct page.